networkx/linalg/laplacianmatrix.py

```diff
@@ -272,6 +273,10 @@ def directed_combinatorial_laplacian_matrix(
     """
     from scipy.sparse import spdiags, linalg
 
+    warnings.warn(
```

Comment:
Don't we still need to figure out whether we are using ``scipy.sparse``, ``pysparse``, or ``scipy.csgraph`` first?

Reply:
Yes, we do -- but we were working through one function for this in the sprint and decided to make them consistent first and then once we have decided which way to proceed we could update it. Also, we might want to eventually make the type controllable too...???... Do we?

We actually had some tricky parts today due to scipy.sparse matrices turning into numpy matrices when we didn't expect them to.  That could lead to memory bottlenecks.. and unexpected types too.